src/test/munit/trigger-test-case.xml

```diff
@@ -184,13 +183,14 @@
         </munit:behavior>
 
         <munit:execution>
-            <until-successful maxRetries="100" millisBetweenRetries="500">
+            <until-successful maxRetries="20" millisBetweenRetries="500">
                 <munit-tools:retrieve key="messages"/>
                 <validation:validate-size value="#[payload]" min="2" max="2"/>
             </until-successful>
         </munit:execution>
 
         <munit:validation>
+            <munit-tools:retrieve key="messages"/>
```

Comment:
check if we are not hiding a runtime bug with this

Reply:
It seems that the test was buggy because if you put a "sleep" before the "until-succesful" statement the test starts to fail at older runtime versions too. So I think that a change at the runtime revealed this bug.

Reply:
It seems that the test was buggy because if you put a "sleep" before the "until-succesful" statement the test starts to fail at older runtime versions too. So I think that a change at the runtime revealed this bug.